pkg/networkmanager/org.cockpit-project.cockpit-networkmanager.metainfo.xml.in

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.cockpit_project.cockpit_networkmanager</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Networking</name>
+  <summary>Cockpit configuration of NetworkManager and Firewalld</summary>
+  <description>
+    <p>
+    	This tool manages networking such as bonds, bridges, teams, VLANs
+    	and firewalls using NetworkManager and Firewalld.
+        NetworkManager is incompatible with Ubuntu's default systemd-networkd and
+	Debian's ifupdown scripts. 
```

Comment:
So with NetworkManager mentioned in the <summary> should the warning be here?

Reply:
Yes, it should. thanks!